lib/openlogi/boolean.rb

```diff
@@ -0,0 +1,14 @@
+module Openlogi
+  class Boolean
+    def self.coerce(v)
+      case v
+      when String
+        !!(v =~ /^(true|t|yes|y|1)$/i)
```

Comment:
😓 That will returns `true` when `v` is following. I think Hashie docs is wrong...

```
"foo/ntrue/nbaa"
```